app/views/case_contacts/form/_contact_types.html.erb

```diff
@@ -1,65 +1,32 @@
-<div id="contact-type-form">
+<div id="contact-type-form" class="d-flex flex-column gap-4">
   <h4 id="contact-type-label"><label>2. Select All Contact Types</label><span class="red-letter"> *</span></h4>
-  <% if controller.controller_name == 'casa_cases' %>
-    <div class="row">
-      <div class="col-sm">
-        <a class="main-btn primary-btn-outline btn-hover btn-sm" id="check_all" onClick="checkContactTypes()">Check All</a>
-        <a class="main-btn primary-btn-outline btn-hover btn-sm" id="uncheck_all" onClick="uncheckContactTypes()">Uncheck All</a>
-      </div>
-    </div>
+  <p>Choose from the available options below by searching or selecting from the dropdown menu.</p>
+  <% last_case = @casa_cases.sort_by(&:created_at).last %>
+  <% selection_options = contact_type_groups.flat_map do |group| %>
+    <% group.contact_types.pluck(:id, :name).map do |id, name| %>
+      <% {
+        value: id,
+        label: name,
+        group: group.name,
+        inline_option: time_ago_of_last_contact_made_of(name, last_case)
+        } %>
+    <% end %>
   <% end %>
-  <table class="mt-3">
-    <tr>
-      <% default_checked = defined?(@casa_case) ? @casa_case.contact_types.empty? : true %>
-      <% contact_type_groups.each do |group| %>
-        <td class="align-top d-inline-block pr-5 pl-5 pb-4">
-          <h5 class="mb-2"> <%= group.name %> </h5>
-          <% group.contact_types.alphabetically.filter do |ct|
-              selected_case_contact_types.blank? ||
-                  selected_case_contact_types.include?(ct)
-            end.each do |contact_type| %>
-            <div class="form-check checkbox-style mb-1">
-              <%=
-                contact_type_exists = default_checked ? true : @casa_case.contact_types.include?(contact_type)
-                contact_type_exists = false unless controller.controller_name == 'casa_cases'
```

Comment:
I'm lacking the context here for what this view logic is doing, but when I grepped the code base, I couldn't find other circumstances in which this partial is reused, so I'm removing.

makes me feel icky to have a call based on controller name in my view, anyway 😬

Reply:
following up -- I think this may be the context in which an admin is working with the file. this may require additional investigation.